python/lsst/pipe/tasks/dcrAssembleCoadd.py

```diff
@@ -796,7 +805,9 @@ def fillCoadd(self, dcrModels, skyInfo, tempExpRefList, weightList, calibration=
             the model for one subfilter.
         """
         dcrCoadds = []
+        refModel = dcrModels.getReferenceImage()
         for model in dcrModels:
+            model.array = (model.array - refModel)*self.config.accelerateModel + refModel
```

Comment:
Should this be conditional on `accelerateModel > 1`, like the similar code in `buildMatchedTemplate`?